www/install/steps/step6.php

```diff
@@ -85,7 +87,14 @@
             </td>
         </tr>
         <tr>
-            <td class='formlabel'>"._('Root password')."</td>
+            <td class='formlabel'>"._('User with root privileges').$star."</td>
```

Comment:
Did you add : _('User with root privileges')  to the gettext file ? 
Please add the gettext reference file too